egs2/TEMPLATE/asr1/db.sh

```diff
@@ -41,6 +41,8 @@ LIBRISPEECH=
 LIBRILIGHT_LIMITED=
 FSC=
 SLURP=
+SLURP_S=
+LIBRITRANS_S=
```

Comment:
It is better to add a comment `# Output file path` at the end of the lines if they are expected to be an output directory instead of a well-prepared one.